.github/scripts/trymerge.py

```diff
@@ -1669,7 +1669,19 @@ def get_classifications(
     # going forward. It's preferable to try calling Dr.CI API directly first
     # to get the latest results as well as update Dr.CI PR comment
     drci_classifications = get_drci_classifications(pr_num=pr_num, project=project)
-    print(f"From Dr.CI API: {json.dumps(drci_classifications)}")
+
+    def get_readable_drci_results(drci_classifications: Any) -> str:
+        try:
+            s = f"From Dr.CI API ({pr_num}):\n"
+            for classification, jobs in drci_classifications.items():
+                s += f"  {classification}: \n"
+                for job in jobs:
+                    s += f"    {job['id']} {job['name']}\n"
+            return s
+        except Exception:
+            return f"From Dr.CI API: {json.dumps(drci_classifications)}"
+
+    print(get_readable_drci_results(drci_classifications))
```

Comment:
This part just makes the Dr.CI results nicer to read, I can remove this if there are objections
Old:
```
From Dr.CI API: {"FAILED": [], "FLAKY": [], "BROKEN_TRUNK": [{"workflowId": 5859347753, "id": 15885841715, "name": "pull / linux-bionic-cuda12.1-py3.10-gcc9-sm86 / test (default, 4, 5, linux.g5.4xlarge.nvidia.gpu)", "jobName": "linux-bionic-cuda12.1-py3.10-gcc9-sm86 / test (default, 4, 5, linux.g5.4xlarge.nvidia.gpu)", "conclusion": "failure", "completed_at": "2023-08-14T19:50:09Z", "html_url": "https://github.com/pytorch/pytorch/actions/runs/5859347753/job/15885841715", "head_branch": "gh/mrshenli/385/head", "pr_number": 107160, "head_sha": "1abba7bb35f1562329d0235dedf171790cd162a5", "failure_captures": ["inductor/test_torchinductor.py::TritonCodeGenTests::test_optimize_indexing_dtype_with_constraint"], "failure_line": "FAILED [0.1966s] inductor/test_torchinductor.py::TritonCodeGenTests::test_optimize_indexing_dtype_with_constraint - torch._dynamo.exc.BackendCompilerFailed: backend='inductor' raised:", "time": "2023-08-14T19:00:56.524425Z"}], "UNSTABLE": [{"workflowId": 5860028213, "id": 15887855756, "name": "trunk / linux-focal-rocm5.6-py3.8 / test (default, 1, 3, linux.rocm.gpu, unstable)", "jobName": "linux-focal-rocm5.6-py3.8 / test (default, 1, 3, linux.rocm.gpu, unstable)", "conclusion": "failure", "completed_at": "2023-08-14T22:04:13Z", "html_url": "https://github.com/pytorch/pytorch/actions/runs/5860028213/job/15887855756", "head_branch": "ciflow/trunk/107160", "pr_number": 107160, "head_sha": "1abba7bb35f1562329d0235dedf171790cd162a5", "failure_captures": ["inductor/test_torchinductor_opinfo.py::TestInductorOpInfoCUDA::test_comprehensive_tanh_cuda_float16"], "failure_line": "FAILED [0.0914s] inductor/test_torchinductor_opinfo.py::TestInductorOpInfoCUDA::test_comprehensive_tanh_cuda_float16", "time": "2023-08-14T20:14:59.760637Z"}, {"workflowId": 5860028213, "id": 15887855946, "name": "trunk / linux-focal-rocm5.6-py3.8 / test (default, 2, 3, linux.rocm.gpu, unstable)", "jobName": "linux-focal-rocm5.6-py3.8 / test (default, 2, 3, linux.rocm.gpu, unstable)", "conclusion": "failure", "completed_at": "2023-08-14T22:10:08Z", "html_url": "https://github.com/pytorch/pytorch/actions/runs/5860028213/job/15887855946", "head_branch": "ciflow/trunk/107160", "pr_number": 107160, "head_sha": "1abba7bb35f1562329d0235dedf171790cd162a5", "failure_captures": ["profiler/test_profiler.py::TestProfiler::test_kineto_profiler_with_environment_variable"], "failure_line": "FAILED [6.9686s] profiler/test_profiler.py::TestProfiler::test_kineto_profiler_with_environment_variable - AssertionError: False is not true : Kineto is not working properly with the Dynolog environment variable", "time": "2023-08-14T20:15:00.814152Z"}, {"workflowId": 5860028213, "id": 15887856086, "name": "trunk / linux-focal-rocm5.6-py3.8 / test (default, 3, 3, linux.rocm.gpu, unstable)", "jobName": "linux-focal-rocm5.6-py3.8 / test (default, 3, 3, linux.rocm.gpu, unstable)", "conclusion": "failure", "completed_at": "2023-08-14T23:02:38Z", "html_url": "https://github.com/pytorch/pytorch/actions/runs/5860028213/job/15887856086", "head_branch": "ciflow/trunk/107160", "pr_number": 107160, "head_sha": "1abba7bb35f1562329d0235dedf171790cd162a5", "failure_captures": ["inductor/test_torchinductor_opinfo.py::TestInductorOpInfoCUDA::test_comprehensive_nn_functional_instance_norm_cuda_float32"], "failure_line": "FAILED [0.6190s] inductor/test_torchinductor_opinfo.py::TestInductorOpInfoCUDA::test_comprehensive_nn_functional_instance_norm_cuda_float32", "time": "2023-08-14T20:15:00.976758Z"}]}
```
New:
```
From Dr.CI API (107160):
  FAILED: 
  FLAKY: 
  BROKEN_TRUNK: 
    15885841715 pull / linux-bionic-cuda12.1-py3.10-gcc9-sm86 / test (default, 4, 5, linux.g5.4xlarge.nvidia.gpu)
  UNSTABLE: 
    15887855756 trunk / linux-focal-rocm5.6-py3.8 / test (default, 1, 3, linux.rocm.gpu, unstable)
    15887855946 trunk / linux-focal-rocm5.6-py3.8 / test (default, 2, 3, linux.rocm.gpu, unstable)
    15887856086 trunk / linux-focal-rocm5.6-py3.8 / test (default, 3, 3, linux.rocm.gpu, unstable)
```

These are not the same example sorry